include/map.h

```diff
@@ -135,8 +148,8 @@ typedef struct StaticNPC {
     /* 0x028 */ s8 dropFlags;
     /* 0x029 */ s8 itemDropChance; // %
     /* 0x02A */ ItemDrop itemDrops[8];
-    /* 0x05A */ StatDrop heartDrops[8];
-    /* 0x09A */ StatDrop flowerDrops[8];
+    /* 0x05A */ StatDrop heartDrops[5];
```

Comment:
Oh, this branch isn't matching because you changed the size of this struct

Reply:
Whoops 2